DECISION-RECORD.md

```diff
@@ -0,0 +1,292 @@
+# Trailblaze (September 2020)
+
+TBA
+
+# Prototyping (July 2020)
+
+Research and decision record from the July 2020 prototyping week.
+
+## /entity/item/Q42 vs /entity/Q42
+
+**Decision: /entity/item/Q42 TBA why...**
+
+---
+
+## Consistent "default edit parameters"
+
+**Decision: New edit endpoints should all equally implement certain “default edit parameters” (e.g. token, bot, baserevid, summary, tags).**
```

Comment:
I think we didn't actually implemented them in the spec yet

Reply:
We have done summary and tags.
I'll be doing baserevid and bot today.
token we should remove now per the other PR.